RELEASE/scripts/sl_ascend/sl_mr2017.ash

```diff
@@ -1433,6 +1433,48 @@ int horseCost()
 	return 0;
 }
 
+string horseNormalize(string horseText)
+{
+	switch(horseText)
+	{
+		case "normal":	   return "normal";
+		case "regen":	   return "normal";
+		case "init":	   return "normal";
+		case "dark":	   return "dark";
+		case "meat":	   return "dark";
+		case "-combat":	   return "dark";
+		case "noncombat":  return "dark";
+		case "non-combat": return "dark";
+		case "crazy":	   return "crazy";
+		case "hookah":	   return "crazy";
+		case "random":	   return "crazy";
+		case "pale":	   return "pale";
+		case "res":	   return "pale";
+		case "resistance": return "pale";
+		case "spooky":	   return "pale";
+		case "damage":	   return "pale";
+		case "return":	   return "return";
+		case "":	   return "return";
+	}
+
+	if (contains_text(horseText, "normal horse"))
```

Comment:
I had put these here because `getHorse` has it in some cases. But it seems like the literal value "dark horse" is always used in `_horsery`, and maybe the others are too. In which case these could go into the `switch`, instead.

Need to test it with all the different horse types first, though.